kemp-loadmaster-ha-pair/azuredeploy.json

```diff
@@ -0,0 +1,359 @@
+{
+   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion": "1.0.0.0",
+   "parameters": {
+      "balPassword": {
+         "type": "securestring",
+         "metadata": {
+            "description": "Password for bal account"
+         }
+      }
+   },
+   "variables": {
+      "availabilitySetName": "AvSet",
+      "vmNamePrefix": "KEMPVLM",
+      "loadBalancerName": "AzureLB",
+      "adminUsername": "bal",
```

Comment:
use a parameter for this, increases surface area if everyone has the same user name...